apps/prairielearn/src/pages/courseSyncs/courseSyncs.sql

```diff
@@ -6,14 +6,10 @@ SELECT
 FROM
   job_sequences AS js
   JOIN pl_courses AS c ON (c.id = js.course_id)
-  JOIN users AS u on (u.user_id = js.user_id)
+  LEFT JOIN users AS u on (u.user_id = js.user_id)
```

Comment:
You probably want a corresponding change in the `.ejs` file to render something sensible ("System"?) in place of a null UID.

Reply:
Good point. Is "System" a good option? Is there any other situation where the user would be null? Would "Unknown" be generic and informative enough?

Reply:
We've been using "System" in PT and it seems to give the right impression (we do know how this was triggered, it just wasn't by a regular user action).